src/main/java/com/ibm/cloud/cloudant/kafka/schema/ConnectRecordMapper.java

```diff
@@ -33,6 +33,9 @@
     private static Logger LOG = LoggerFactory.getLogger(ConnectRecordMapper.class);
 
     public Map<String, Object> apply(ConnectRecord<R> record) {
+        if (record.value() == null) {
+            throw new IllegalArgumentException("Record value is null");
```

Comment:
The behaviour we [describe in the README suggests](https://github.com/IBM/cloudant-kafka-connector/blob/main/README.md#sink) that we'll create an empty document for a tombstone if it isn't filtered out. So perhaps we should just return an empty Map here rather than throwing an exception?

I think we don't want to always discard tombstones because _eventually_ it would be good to actually handle deletions (somehow).

If we are going to throw here I think it ought to be a `DataException` that suggests filtering tombstones along with a README update, but I'm not really sure which is more useful behaviour.